### PR TITLE
Add project 6

### PR DESCRIPTION
**Pull request by matthewlang**
Here's the file format drawing project!

### CODE REVIEW

**Comment by pkirlin** [comment]
picture2
picture1
panda
lynx
homer

Here are updated versions of the pictures --- it's not critical, but each one has a border around it now, which is what the students will see in their output.

File: projects/project06/P6.ipynb [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+   "execution_count": null,
+   "metadata": {
+    "deletable": false,
+    "editable": false
+   },
+   "outputs": [],
+   "source": [
+    "# Run this cell to log in to okpy.org so that you can submit\n",
+    "# at the end of the lab.\n",
+    "from cs1.notebooks import *\n",
```
[the remark this answers is not in the document]
Comment (pkirlin) [reply projects/project06/P6.ipynb:242]: I pasted in the solution code into the notebook to see how it runs, and it works great!  The only issue is that the graphics drawing code complains if you call set_line_thickness before the canvas is open.  So we may want to add a note about that in the project, telling the students to make sure the canvas is open before doing the line thickness operation.

also, I made new versions of the output pictures with the border that the graphics library now adds automatically.  See comment below.

File: projects/project06/P6.ipynb [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+   "source": [
+    "## Writing your program\n",
+    "\n",
+    "The file `draw.py` is in the same folder as this notebook.\n",
+    "\n",
+    "Your `main()` function should:\n",
+    "* Prompt the user for the name of a file to open.\n",
+    "* Open the file and read the size of the canvas to open.\n",
+    "* Using a loop, read and execute the commands, calling `draw_line` from the graphics library ([reference](https://matthewlang.github.io/comp141/graphics.html) and `draw_dotted_line` (you will have to write this function).\n",
+    "* Display the canvas by calling `display_canvas`.\n",
```
[the remark this answers is not in the document]
Comment (matthewlang) [reply projects/project06/P6.ipynb:113]: Good catch!

File: projects/project06/P6.ipynb (hunk 1 above)
[the remark this answers is not in the document]
Comment (matthewlang) [reply projects/project06/P6.ipynb:242]: Thank you! Using them!

**Comment by matthewlang** [comment]
> Looks good. A few minor nits.

Thanks!

**Pull request merged**